src/app/pages/test-results-search/components/advanced-search/advanced-search.ts

```diff
@@ -70,6 +119,15 @@ export class AdvancedSearchComponent {
     }],
   };
 
+  populateActivityArray() {
+    const activityCodeArray: any[] = [];
+    for (let i = 0; i < activityCodeModelList.length; i += 1) {
+      activityCodeArray.push(`${activityCodeModelList[i].activityCode} - ${activityCodeModelList[i].description}`);
+    }
+    activityCodeArray.unshift('All');
+    return activityCodeArray;
+  }
```

Comment:
I don't think we need to do this - we already have an array of codes/descriptions of the codes, believe we could just do something like this
```
  activityCodes: { activityCode: string; description: string; }[] = [
    { activityCode: 'All', description: 'All' },
    ...activityCodeModelList,
  ];
```